site/app/templates/grading/simple/Display.twig

```diff
@@ -15,9 +15,9 @@
         </div>
     {% endif %}
 
-    <h2>
+    <h1>
         {{ gradeable.getTitle() }}
-    </h2>
+    </h1>
     <p>
```

Comment:
This seems to be breaking the travis tests.
Change the "h2" in the click_nav_grade_button call in test_simple_grader (in the run_tests method) to an "h1".

Reply:
Yup, that was it!  Thanks!